pytorch_lightning/core/lightning.py

```diff
@@ -17,7 +17,7 @@
 from pytorch_lightning.core.grads import GradInformation
 from pytorch_lightning.core.hooks import ModelHooks
 from pytorch_lightning.core.memory import ModelSummary
-from pytorch_lightning.core.saving import ModelIO, load_hparams_from_tags_csv, load_hparams_from_yaml
+from pytorch_lightning.core.saving import ModelIO, PRIMITIVE_TYPES, ALLOWED_CONFIG_TYPES
```

Comment:
Generally speaking, I don't  feel like core.saving is the right place for something like ALLOWED_CONFIG_TYPES.
It feels more fundamental than this.